extra-resources/mods/Mythos/chinese.cardset

```diff
@@ -170,9 +155,9 @@ card('GUAN YU') {
         summon 2 of 'Terracotta Soldier' to 'you' zone 'Hand'
     }
 }
-// Info: http://en.wikipedia.org/wiki/Eight_Immortals
-// Image: http://upload.wikimedia.org/wikipedia/commons/9/9a/Eight_Immortals_Crossing_the_Sea_-_Project_Gutenberg_eText_15250.jpg
-// License: Public Domain
+// https://en.wikipedia.org/wiki/Eight_Immortals
+// https://commons.wikimedia.org/wiki/File:Eight_Immortals_Crossing_the_Sea_-_Project_Gutenberg_eText_15250.jpg
+// Exception: <class 'selenium.common.exceptions.NoSuchElementException'>
```

Comment:
OK I think I got all of them, incidentally happens I forgot to search the Chinese file for exceptions in my IDE, but did for the other 3. Go figure.